### DISCUSSION TITLE
Testing with React Native

### DISCUSSION THREAD

**Discussion by kaili-kameoka** [jaredpalmer/formik#3383] [Q&A]
What is the correct way to test Formik in React Native. I can't get the Blur event to fire in my tests. It seems like the blurHandler is loking for a persist property from the web version that doesn't exist in the Native version. 
Here is a link to a repository: https://github.com/kaili-kameoka/react-native-sandbox/tree/formik

**Comment by justjapann** [comment]
you have to pass a () inside the handleBlur, for example onBlur={handleBlur('email')}, or you can join it to formik, Ex: onBlur={formik.handleBlur}
